src/pgp.rs

```diff
@@ -457,7 +457,7 @@ mod tests {
         let mut keyring = Keyring::new();
         keyring.add(KEYS.alice_public.clone());
         keyring.add(KEYS.bob_public.clone());
```

Comment:
this is using a different executor on purpose, it is not unnecessary

Reply:
at least it was when I wrote it

Reply:
I don't get it. Is there any difference between using https://docs.rs/async-std/1.10.0/async_std/task/fn.block_on.html and https://docs.rs/futures-lite/1.12.0/futures_lite/future/fn.block_on.html?

Reply:
I get that they're different executors, but why does that need to be?  IIUC the first test to access these globals will execute, with the old code starting a new executor which presumably is never stopped.  With the new code it'd use the same executor as the test is being run in (which is torn down at the end of the test).  In either case, once the value is computed all other tests access it from there.  I don't think there's any downside to computing this on one of the test's executors?